bcreg-aca/config/services.yml

```diff
@@ -310,10 +311,11 @@ issuers:
 
       - schema: business_number.registries.ca
         issuer_url: $APPLICATION_URL_ADDRESS
-        description: TODO
         details:
-          label_en: Corporate Business Number
           url_en: $APPLICATION_URL_ADDRESS
+          highlighted_attributes:
+            - business_number
+          credential_title: Business Number
```

Comment:
Credential title should always be referring to an attribute key, in this case it would be `business_number`